osbuild/mounts.py

```diff
@@ -123,8 +123,10 @@ def __init__(self, args):
         self.mountpoint = None
         self.check = False
 
-    @abc.abstractmethod
-    def translate_options(self, options: Dict) -> List:
+    @staticmethod
+    def translate_options(options: Dict) -> List:
+        if options.get("readonly", False):
+            return ["-o", "ro"]
         return []
```

Comment:
By dropping `self` from the super-class, you prevent any derived function from using it. I don't think this is what we want. I would keep this as a normal method-call with `self`, and just annotate it as unused for pylint.

Reply:
I had that at first, but changed to a static method for pylint.